libhb/hb_dict.c

```diff
@@ -139,7 +139,7 @@ static hb_value_t* xform_bool(const hb_value_t *value, hb_value_type_t type)
             return json_real(b);
         case HB_VALUE_TYPE_STRING:
         {
-            char *s = hb_strdup_printf("%"JSON_INTEGER_FORMAT, b);
```

Comment:
Isn't JSON_INTEGER_FORMAT supposed to be the right formatter for json_int_t in jansson?

Reply:
I think the reason I added this was that `JSON_INTEGER_FORMAT` is defined as `I64d` when compiling for Windows, and the mingw cross-compiler doesn't recognize that as a long long int (although there's no warning in this particular case because of [this definition](https://github.com/HandBrake/HandBrake/blob/96cc7f80bd42299a6deed334a0ee58094603daf6/libhb/handbrake/common.h#L38). So I changed it to `lld` which is also supported by MSVCRT and doesn't give a warning. But as the warning is actually erroneous, changing it back won't be a problem.

Reply:
I feel like it would be better to fix the define in jansson library, but I guess not many are compiling it with a 32 bit json_int_t.

Reply:
On reflection, I think it would be better to take this out for now, and possibly include it in a cleanup of libhb code at some point.